ci/release/update-version.sh

```diff
@@ -39,3 +39,4 @@ for FILE in conda/environments/*.yml; do
 done
 
 sed_runner "s/export UCX_PY_VERSION=.*/export UCX_PY_VERSION='${NEXT_UCX_PY_VERSION}'/g" ci/gpu/build.sh
+sed_runner "s/export UCX_PY_VERSION=.*/export UCX_PY_VERSION='${NEXT_UCX_PY_VERSION}'/g" ci/cpu/build.sh
```

Comment:
isn't this a duplicate of the line?

Reply:
almost. one is `cpu/build.sh` the other is `gpu/build.sh`

Reply:
This one is `cpu` and the previous one is `gpu`